code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -208,28 +210,28 @@
 				return
 			wires = new /datum/wires/explosive/pizza(src)
 			bomb = I
-			to_chat(user, span_notice("You put [I] in [src]. Sneeki breeki..."))
+			balloon_alert(user, "bomb placed")
 			update_appearance()
 			return
 		else if(bomb)
-			to_chat(user, span_warning("[src] already has a bomb in it!"))
+			balloon_alert(user, "already rigged!")
 	else if(istype(I, /obj/item/pen))
 		if(!open)
 			if(!user.can_write(I))
 				return
-			var/obj/item/pizzabox/box = boxes.len ? boxes[boxes.len] : src
+			var/obj/item/pizzabox/box = length(boxes) ? boxes[length(boxes)] : src
 			box.boxtag += tgui_input_text(user, "Write on [box]'s tag:", box, max_length = 30)
 			if(!user.canUseTopic(src, be_close = TRUE))
 				return
-			to_chat(user, span_notice("You write with [I] on [src]."))
+			balloon_alert(user, "writing box tag...")
 			boxtag_set = TRUE
 			update_appearance()
 			return
 	else if(is_wire_tool(I))
 		if(wires && bomb)
 			wires.interact(user)
 	else if(istype(I, /obj/item/reagent_containers/cup))
-		to_chat(user, span_warning("That's not a pizza!"))
+		balloon_alert(user, "not a pizza!")
```

Comment:
I think this code is here because pizza used to be a subtype of reagent containers. You should remove it honestly.